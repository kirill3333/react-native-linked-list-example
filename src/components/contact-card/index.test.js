import React from 'react';
import renderer from 'react-test-renderer';

import ContactCard from './index';

it('renders correctly', () => {
  const tree = renderer
    .create(
      <ContactCard
        height={500}
        contact={{
          name: 'John Doe',
          role: 'CEO',
          about: 'Test test test',
        }}
      />,
    )
    .toJSON();
  expect(tree).toMatchSnapshot();
});
