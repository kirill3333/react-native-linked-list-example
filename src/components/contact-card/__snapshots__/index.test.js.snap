// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<View
  style={
    Array [
      Object {
        "flex": 1,
      },
      Object {
        "height": 500,
      },
    ]
  }
>
  <View
    style={
      Object {
        "flexDirection": "row",
        "justifyContent": "center",
        "paddingTop": 20,
      }
    }
  >
    <Text
      style={
        Object {
          "fontSize": 22,
          "fontWeight": "bold",
        }
      }
    >
      John
    </Text>
    <Text>
       
    </Text>
    <Text
      style={
        Object {
          "fontSize": 22,
          "fontWeight": "normal",
        }
      }
    >
      Doe
    </Text>
  </View>
  <View
    style={
      Object {
        "flexDirection": "row",
        "justifyContent": "center",
        "paddingTop": 5,
      }
    }
  >
    <Text
      style={
        Object {
          "color": "#828282",
          "fontSize": 16,
        }
      }
    >
      CEO
    </Text>
  </View>
  <View
    style={
      Object {
        "flexDirection": "column",
        "justifyContent": "flex-start",
        "paddingHorizontal": 20,
        "paddingTop": 20,
      }
    }
  >
    <Text
      style={
        Object {
          "fontSize": 16,
          "fontWeight": "bold",
        }
      }
    >
      About me
    </Text>
    <Text
      style={
        Object {
          "color": "#828282",
          "fontSize": 16,
          "lineHeight": 18,
          "paddingTop": 10,
        }
      }
    >
      Test test test
    </Text>
  </View>
</View>
`;
